src/material.h

```diff
@@ -29,6 +30,9 @@ class material_type
         int _fire_resist;
         int _chip_resist;       // Resistance to physical damage of the item itself
         int _density;   // relative to "powder", which is 1
+        bool _edible;
+
+        mutable std::map<vitamin_id, double> _vitamins;
```

Comment:
> This mutable looks like a big warning that something is not done right but being hacked around instead.

It's there because `operator[]` requires it in the `vitamin()` accessor. That variable is `mutable` as the appearance external to the class is of a `const` variable. This is the correct and intended use for that modifier.